[plugin.video.tv3.cat] 1.4.8

jqandreu opened the pull request:
### Description
<!--- Provide a short summary of submitted add-on in case it's a new addition. -->
<!--- If it's plugin update only highlight biggest changes if needed. -->
<!--- Make sure you follow the checklist below before finalizing your pull-request. -->
Adapted to Kodi 19.0 and fixed errors
### Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply like this: [X] -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x ] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x ] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

enen92 commented:
Please have a look at travis failures, you have quite a bit of changes ahead before anyone actually review the code:
https://travis-ci.com/github/xbmc/repo-plugins/builds/218838825

enen92 commented on plugin.video.tv3.cat/addon.xml:
```diff
@@ -0,0 +1,28 @@
+        <source>https://github.com/jqandreu/plugin.video.tv3.cat</source>
+        <website>http://www.tv3.cat</website>
+    </extension>
+    <assets>
```
assets is a child of metadata not root/addon

enen92 replied:
You have to delete the assets that is outside of the metadata. Right now you have multiple copies

jqandreu replied:
Fixed. Thanks

enen92 commented on plugin.video.tv3.cat/resources/language/Catalan/strings.po:
```diff
@@ -0,0 +1,125 @@
+# Kodi Media Center language file
```
move to: resources/language/resource.language.ca_es/strings.po

enen92 commented on plugin.video.tv3.cat/resources/language/English/strings.po:
```diff
@@ -0,0 +1,125 @@
+# Kodi Media Center language file
```
move to: resources/language/resource.language.en_gb/strings.po

enen92 commented on plugin.video.tv3.cat/resources/language/German/strings.po:
```diff
@@ -0,0 +1,125 @@
+# Kodi Media Center language file
```
move to: resources/language/resource.language.de_de/strings.po

enen92 commented on plugin.video.tv3.cat/resources/language/Norwegian/strings.po:
```diff
@@ -0,0 +1,125 @@
+# Kodi Media Center language file
```
move to: resources/language/resource.language.nb_no/strings.po

enen92 commented on plugin.video.tv3.cat/resources/language/Spanish/strings.po:
```diff
@@ -0,0 +1,125 @@
+# Kodi Media Center language file
```
move to: resources/language/resource.language.es_es/strings.po

enen92 commented on plugin.video.tv3.cat/addon.xml:
```diff
@@ -0,0 +1,28 @@
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <summary lang="en">Entertainment, news, documentaries, etc from catalan television www.tv3.cat</summary>
```
en_GB

enen92 commented on plugin.video.tv3.cat/addon.xml:
```diff
@@ -0,0 +1,28 @@
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <summary lang="en">Entertainment, news, documentaries, etc from catalan television www.tv3.cat</summary>
+        <description lang="en">Entertainment, news, documentaries, etc from catalan television www.tv3.cat. [CR]Featured shows, most viewed, A to Z programs. [CR]Live channels: TV3cat, K33, Super3, News 3/24, Esport3</description>
```
en_GB

enen92 commented on plugin.video.tv3.cat/addon.xml:
```diff
@@ -0,0 +1,28 @@
+        <summary lang="ca">Tots els programes de TV3 a la carta.</summary>
+        <description lang="ca">Tota la programació de TV3 a la carta. [CR]Les llistes dels més vistos, destacats, programes ordenats per ordre alfabètic. [CR]Els canals en directe: TV3cat, K33, Super3, 3/24 i Esport3 </description>
+        <summary lang="es">Toda la programación de TV3 a la carta</summary>
+        <description lang="es">Toda la programación de TV3 a la carta.[CR]Las listas de los más vistos, destacados, programas ordenados de la A a la Z. [CR]Canales en directo: TV3cat, K33, Super3, 3/24, Esport3</description>
```
es_ES

enen92 commented on plugin.video.tv3.cat/addon.xml:
```diff
@@ -0,0 +1,28 @@
+        <description lang="en">Entertainment, news, documentaries, etc from catalan television www.tv3.cat. [CR]Featured shows, most viewed, A to Z programs. [CR]Live channels: TV3cat, K33, Super3, News 3/24, Esport3</description>
+        <summary lang="ca">Tots els programes de TV3 a la carta.</summary>
+        <description lang="ca">Tota la programació de TV3 a la carta. [CR]Les llistes dels més vistos, destacats, programes ordenats per ordre alfabètic. [CR]Els canals en directe: TV3cat, K33, Super3, 3/24 i Esport3 </description>
+        <summary lang="es">Toda la programación de TV3 a la carta</summary>
```
es_ES

enen92 commented on plugin.video.tv3.cat/addon.xml:
```diff
@@ -0,0 +1,28 @@
+        <platform>all</platform>
+        <summary lang="en">Entertainment, news, documentaries, etc from catalan television www.tv3.cat</summary>
+        <description lang="en">Entertainment, news, documentaries, etc from catalan television www.tv3.cat. [CR]Featured shows, most viewed, A to Z programs. [CR]Live channels: TV3cat, K33, Super3, News 3/24, Esport3</description>
+        <summary lang="ca">Tots els programes de TV3 a la carta.</summary>
```
ca_ES

enen92 commented on plugin.video.tv3.cat/addon.xml:
```diff
@@ -0,0 +1,28 @@
+        <summary lang="en">Entertainment, news, documentaries, etc from catalan television www.tv3.cat</summary>
+        <description lang="en">Entertainment, news, documentaries, etc from catalan television www.tv3.cat. [CR]Featured shows, most viewed, A to Z programs. [CR]Live channels: TV3cat, K33, Super3, News 3/24, Esport3</description>
+        <summary lang="ca">Tots els programes de TV3 a la carta.</summary>
+        <description lang="ca">Tota la programació de TV3 a la carta. [CR]Les llistes dels més vistos, destacats, programes ordenats per ordre alfabètic. [CR]Els canals en directe: TV3cat, K33, Super3, 3/24 i Esport3 </description>
```
ca_ES

jqandreu replied:
Fixed all errors. Thanks

enen92 commented on plugin.video.tv3.cat/addon.xml:
```diff
@@ -0,0 +1,28 @@
+        <summary lang="es">Toda la programación de TV3 a la carta</summary>
+        <description lang="es">Toda la programación de TV3 a la carta.[CR]Las listas de los más vistos, destacados, programas ordenados de la A a la Z. [CR]Canales en directo: TV3cat, K33, Super3, 3/24, Esport3</description>
+        <language>ca</language>
+        <license>GPL-2.0</license>
```
```suggestion
        <license>GPL-2.0-only</license>
```

or 

```suggestion
        <license>GPL-2.0-or-later</license>
```

jqandreu commented:
Fixed all errors and sugestions. The last check also fails due to assets tag in addon.xml. I moved to extension point="xbmc.addon.metadata" Is this correct?

enen92 commented on plugin.video.tv3.cat/resources/lib/tv3cat/DirAZemisio.py:
```diff
@@ -0,0 +1,18 @@
+from resources.lib.video.FolderVideo import FolderVideo
```
dummy empty  `__init__py` files should no longer be needed with python 3

enen92 commented on plugin.video.tv3.cat/resources/lib/tv3cat/TV3Strings.py:
```diff
@@ -0,0 +1,40 @@
+
+    def get(self, string):
+        code = self.strs[string]
+        st = self.addon.getLocalizedString(code).encode("utf-8")
```
shouldn't be needed to encode anything in python3

enen92 reviewed (approved):
Code looks fine, a few minors if you want to adjust but not blockers.

jqandreu commented:
Ok. I will adjust in the next version. Thank you
